package.json

```diff
@@ -9,8 +9,8 @@
     "lint:fix": "prettier --write",
     "start": "react-scripts start",
     "test": "react-scripts test",
-    "test:e2e": "react-scripts test src/tests/e2e",
-    "test:unit": "react-scripts test src/tests/unit"
+    "test:e2e": "react-scripts test src/tests/*.e2e.test.js",
+    "test:unit": "react-scripts test src/components/**/*.spec.js"
```

Comment:
```suggestion
    "test:e2e": "react-scripts test src/**/*.e2e.test.js",
    "test:unit": "react-scripts test src/**/*.spec.js"
```

Reply:
`"test:unit": "react-scripts test src/**/*.spec.js" ` —  doesn't work like that
`  "test:unit": "react-scripts test src/**/**/*.spec.js"` — is ok